owncloudData/src/main/java/com/owncloud/android/data/files/datasources/LocalFileDataSource.kt

```diff
@@ -60,4 +60,5 @@ interface LocalFileDataSource {
     fun saveUploadWorkerUuid(fileId: Long, workerUuid: UUID)
     fun saveDownloadWorkerUuid(fileId: Long, workerUuid: UUID)
     fun cleanWorkersUuid(fileId: Long)
+    fun updateFileWithLastUsage(id: Long?, lastUsage: Long?)
```

Comment:
I would say the ID shouldn't be nullable, as you can see in the other methods. Also, call it `fileId` to keep consistency with the other methods

Reply:
Done